pkgs/development/python-modules/pyxlsb/default.nix

```diff
@@ -0,0 +1,21 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "pyxlsb";
+  version = "1.0.10";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-gGLR6oYm0/GYDosc/pGkSDdHRJJC7LYQE7wt+FQ19oU=";
+  };
+
+  meta = with lib; {
+    description = "Excel 2007-2010 Binary Workbook (xlsb) parser";
+    homepage = "https://github.com/willtrnr/pyxlsb";
+    license = licenses.lgpl3;
```

Comment:
Per [this announcement](https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206) on Discourse, `lgpl3` is deprecated. I see in setup.py that  `license='LGPLv3+'` so this will be `licenses.lgpl3Plus`.